README.rst

```diff
@@ -95,6 +95,10 @@ these commands to update your devstack's domain names:
   tutor dev launch -I --skip-build
   tutor dev stop authoring  # We will run this MFE on the host
 
+* If tutor-mfe is not starting the authoring MFE in development mode (eg. `tutor dev start authoring` fails), it may be due to
+  using a tutor version that expects the MFE name to be frontend-app-course-authoring (the previous name of this repo). To fix
+  this, you can rename the cloned repo directory to frontend-app-course-authoring.
+
```

Comment:
Thanks for documenting this!

Reply:
Actually I guess you can remove this or link to https://discuss.openedx.org/t/repo-rename-frontend-app-course-authoring-frontend-app-authoring/13930/2

Reply:
@bradenmacdonald You mean removing the whole bullet, or the solution?

Reply:
Leave the bullet, just link to that post for the solution.